package.json

```diff
@@ -9,12 +9,12 @@
     "packages/*"
   ],
   "resolutions": {
-    "@patternfly/patternfly": "^5.0.0",
-    "@patternfly/react-code-editor": "^5.0.0",
-    "@patternfly/react-core": "^5.0.0",
-    "@patternfly/react-icons": "^5.0.0",
-    "@patternfly/react-table": "^5.0.0",
-    "@patternfly/react-topology": "^5.0.0",
+    "@patternfly/patternfly": "5.2.0",
+    "@patternfly/react-code-editor": "5.1.0",
```

Comment:
At the moment we can't upgrade `@patternfly/react-code-editor` due to an issue affecting how we load schemas.